src/views/components/share-twitter-button.js

```diff
@@ -0,0 +1,21 @@
+import { h } from 'preact' /** @jsx h */
+
+export default ({ href, text, className }) => {
+  const handleClick = () => {
+    window.open(
+      `https://twitter.com/intent/tweet?text=${encodeURIComponent(text)}&url=${href}`,
```

Comment:
Ótimas escolhas de usar o link direto ao invés de bibliotecas. Como será que fica no mobile? Será que os apps reconhecem automaticamente e já abrem o app? Ou precisamos forçar algo ou passar algum parâmetro?